clients/docs/ccloud.rst

```diff
@@ -0,0 +1,229 @@
+#. .. include:: includes/clients-checkout.rst
+
+#. Change directory to the example for |confluent-cli|.
+
+   .. code-block:: bash
+
+      cd clients/cloud/ccloud/
+
+#. .. include:: includes/client-example-create-file.rst
+
+
+Basic Producer and Consumer
+----------------------------
+
+.. include:: includes/producer-consumer-description.rst
+
+
+Produce Records
+~~~~~~~~~~~~~~~
+
+.. Should steps 1 through 3 below be under this  "Produce Records" section or before it?
+
+#. Create the topic in |ccloud|.
+
+   .. code-block:: bash
+
+      ccloud kafka topic create test1
+
+#. Run the :ref:`Confluent Cloud CLI producer <ccloud_kafka_topic_produce>`,
+   writing messages to topic ``test1``, passing in additional arguments:
+
+   -  ``--parse-key --delimiter ,``: pass key and value, separated by a comma
+
+   .. code-block:: bash
+
+      ccloud kafka topic produce test1 --parse-key --delimiter ,
+
+.. Is there any output the user should verfiy when running the previous command or expected output they should see?
+
+#. Type a few messages, using a ``,`` as the separator between the message key
+   and value:
+
+   .. code-block:: bash
+
+      alice,{"count":0}
+      alice,{"count":1}
+      alice,{"count":2}
+
+#. When you are done, press ``Ctrl-C``.
+
+.. Is there a source code link for this one? I only saw two .sh files in the ccloud directory, but we have
+   basic consumer/producer and then Avro consumer/producer. So assuming we'll need 4 source code links in total for this page?
+
+Consume Records
+~~~~~~~~~~~~~~~
+
+.. The below step orginally said "Confluent Confluent CLI Consumer" originally
+   I changed it to Confluent Cloud CLI consumer as I think that is what was meant?
+
+#. Run the :ref:`Confluent Cloud CLI consumer <ccloud_kafka_topic_consume>`,
+   reading messages from topic ``test1``, passing in the following arguments:
+
+   -  ``-b``: print all messages from the beginning of the topic -
+   - ``--print-key``: print key and value (by default, it only prints value)
+
+   .. code-block:: bash
+
+      ccloud kafka topic consume test1 -b --print-key
+
+   You should see the messages you typed in the previous step.
+
+   .. code:-block: bash
+
+       alice   {"count":0}
+       alice   {"count":1}
+       alice   {"count":2}
+
+#. When you are done, press ``CTRL-C``.
+
+.. Not sure the correct source code link. I only saw two .sh files in the ccloud directory, but we have
+   basic consumer/producer and then Avro consumer/producer. So assuming we'll need 4 source code links in total for this page?
+
+#. View the :devx-examples:`consumer code|clients/cloud/java/src/main/java/io/confluent/examples/clients/cloud/ccloud/ccloud-example.sh`.
+
+
+Avro And Confluent Cloud Schema Registry
+----------------------------------------
+
+.. include:: includes/client-example-schema-registry-3.rst
+
+
+Produce Avro Records
+~~~~~~~~~~~~~~~~~~~~
+
+.. Should steps 1 through 3 below be under this  "Produce Avro Records" section or before it?
+
```

Comment:
@ybyzek Should steps 1 through 3 below stay as is or be put before the "Produce Records" section?

Reply:
Given the addition above, the line below can be removed: (line 125-130).  So the section on Produce Avro Records should start with `Create the topic in |ccloud|.`

Reply:
Updated